generator/osm_source.cpp

```diff
@@ -524,6 +532,30 @@ class MainFeaturesEmitter : public EmitterBase
 };
 }  // anonymous namespace
 
+void SyncOfstream::Open(string const & fullPath)
```

Comment:
Please see my detailed explanation above, near your comment "Please, move this out to the separate class."